src/Components/Events/Cards/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {Text, StyleSheet, View, Linking} from 'react-native';
+import ScaledImage from '../../ScaledImage';
+import { withCard } from './../../../Decorators/Card';
+import Badge from './CardBadge';
+
+const EventCard = ({ props, isOver }) => {
+    return (
+     <View style={styles.card} >
+        <ScaledImage width={286}   source={props.highlights.source} />
+        <Text style={styles.title}>{props.title}</Text>
+        <Badge text={props.date} link={require('./../../../assets/events_and_highlights/calendar.png')}  />
+        <Badge text={props.location} link={require('./../../../assets/events_and_highlights/location.png')} />
```

Comment:
I suggest using Icons instead of images (in `Badge` component)

Reply:
for this we have to to install a dependency can i can do that  like material or others

Reply:
please move it to content js so each card have one source of truth regards the content. Thanks.